frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1149,153 +1530,86 @@ function ReaderFooter:addToMainMenu(menu_items)
                 end
             },
             {
-                text = _("Show footer separator"),
+                text = _("Hide inactive items"),
+                help_text = _([[This option will hide null (or inactive) values from temporarily appearing on the status bar. For example, if the frontlight is 'off' (i.e 0 brightness), no symbols or values will be displayed until the brightness is set to a value >= 1.]]),
```

Comment:
The 0 = off implication here strikes me as an irrelevant technical detail from a highly specific case (assuming it's even true there :-) but for the moment let's just get rid of the remaining null.
```suggestion
                help_text = _([[This option will hide inactive items from appearing on the status bar. For example, if the frontlight is 'off' (i.e 0 brightness), no symbols or values will be displayed until the brightness is set to a value >= 1.]]),
```